src/page-layout/NoticesLayout/components/AllNotices/AllNotices.tsx

```diff
@@ -19,25 +19,42 @@ import styles from "./AllNotices.module.scss";
 
 const cn = classNames.bind(styles);
 
+const NUMBER_PER_PAGE = 6;
+
 export default function AllNotices() {
   const router = useRouter();
-
   const { pathname, query } = router;
 
   const [isFilterOpen, setIsFilterOpen] = useState(false);
   const [filters, setFilters] = useState(query);
 
-  const { data, error, isPending, isError } = useGetAllNotices({ limit: 6, ...router.query });
+  const { data, error, isPending, isError } = useGetAllNotices({ limit: NUMBER_PER_PAGE, ...router.query });
 
   const [currentPage, totalPages, setPage] = usePaginationProps({
     totalDataCount: data?.count ?? 0,
-    itemsPageCount: 6,
+    itemsPageCount: NUMBER_PER_PAGE,
   });
 
   const handleOpen = (state: boolean) => {
     setIsFilterOpen(state);
   };
 
+  const handlePageClick = (page: number) => {
+    setPage(page);
+
+    const offset = (page - 1) * NUMBER_PER_PAGE;
+
+    setFilters((prev) => ({
+      ...prev,
+      offset: `${offset}`,
```

Comment:
이렇게 표현해줘도 되겠네요!
문자열 형변환이 필요했다면 String으로 처리하는게 더 괜찮구요!
```suggestion
      offset,
```